tools/devctr/Dockerfile.aarch64

```diff
@@ -105,10 +105,8 @@ RUN cd "$TMP_POETRY_DIR" \
 RUN mkdir "$TMP_BUILD_DIR" \
     && curl https://sh.rustup.rs -sSf | sh -s -- -y --profile minimal --default-toolchain "$RUST_TOOLCHAIN" \
         && rustup target add aarch64-unknown-linux-musl \
-        && rustup component add clippy \
+        && rustup component add clippy llvm-tools-preview \
         && cd "$TMP_BUILD_DIR" \
```

Comment:
I don't think we need `$TMP_BUILD_DIR` at all in this `RUN` directive anymore, since we're no longer building anything from source here

Reply:
Removed this and more to clean up the Rust install.